src/screens/playing.rs

```diff
@@ -16,13 +17,42 @@ pub(super) fn plugin(app: &mut App) {
     );
 }
 
-fn spawn_level(mut commands: Commands) {
+#[derive(Resource, Asset, Reflect, Clone)]
+pub struct GameplayMusic {
+    #[dependency]
+    handle: Handle<AudioSource>,
+    entity: Option<Entity>,
+}
+
+impl FromWorld for GameplayMusic {
+    fn from_world(world: &mut World) -> Self {
+        let assets = world.resource::<AssetServer>();
+        Self {
+            handle: assets.load("audio/bgm/Fluffing A Duck.ogg"),
+            entity: None,
+        }
+    }
+}
+
+fn spawn_level(mut commands: Commands, mut music: ResMut<GameplayMusic>) {
     commands.add(SpawnLevel);
-    commands.play_bgm(BgmHandles::PATH_GAMEPLAY);
+    music.entity = Some(
+        commands
+            .spawn((
+                AudioBundle {
+                    source: music.handle.clone(),
+                    settings: PlaybackSettings::LOOP,
+                },
+                BackgroundMusic,
+            ))
+            .id(),
+    );
 }
 
-fn stop_bgm(mut commands: Commands) {
-    commands.stop_bgm();
+fn stop_bgm(mut commands: Commands, mut music: ResMut<GameplayMusic>) {
+    if let Some(entity) = music.entity.take() {
+        commands.entity(entity).despawn();
```

Comment:
```suggestion
        commands.entity(entity).despawn_recursive();
```
No real difference in this case, but I consider `despawn` to be a footgun as adding a child is possible from any place due to Bevy's decentralized nature

Reply:
Reasonable!